build/charts/antrea/crds/clusternetworkpolicy.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     - name: v1alpha1
       served: true
       storage: false
+      deprecated: true
+      deprecationWarning: "crd.antrea.io/v1alpha1 ClusterNetworkPolicy is deprecated; use crd.antrea.io/v1beta1 ClusterNetworkPolicy"
```

Comment:
this API (and some other ones) were supposed to be removed in #6162 
I think @luolanzone may have missed some files when updating her PR
I will open a PR to address this

Reply:
If #6238 and #6239 are merged, this is likely to be just a documentation PR, as all deprecated APIs will have been removed.
However, we will be able to use these attributes (`deprecated` / `deprecationWarning`) in the future, which is a good thing.